metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -238,6 +240,7 @@ class MetalsLanguageServer(
       () => userConfig,
       config
     )
+    metalsServerConfig.getAndUpdate(config => config.fromInitParams(params))
```

Comment:
Needed for folding option only.

Reply:
Folding was moved into the presentation compiler, so we need to update the presentation compiler config.